data/upgrades.js

```diff
@@ -2545,7 +2585,17 @@
     ],
     "id": 221,
     "xws": "tieshuttle",
-    "image": "upgrades/Title/tie-shuttle.png"
+    "image": "upgrades/Title/tie-shuttle.png",
+    "grants": [
+      {
+        "type": "slot",
+        "name": "Crew"
+      },
+      {
+        "type": "slot",
+        "name": "Crew"
+      }
+    ]
```

Comment:
Yeah, I thought about adding a "revokes" property on the upgrade as well, but I figured I should make one change at a time.

If you like that approach, though, I can do that with this PR.